scripts/utils.ts

```diff
@@ -189,6 +196,9 @@ export async function updateFunctionsMD({ packages, functions }: PackageIndexes)
 
   mdAddons = replacer(mdAddons, addons, 'ADDONS_LIST')
 
+  if (isWindows)
+    mdAddons = mdAddons.replace(/\r\n/g, '\n')
```

Comment:
We don't need to check isWindows, we could always replace them

Reply:
done